quick.html

```diff
@@ -26,36 +26,40 @@
-as Selenium 1. Selenium RC was the most prominent Selenium tool
-before the advent of Selenium WebDriver. Selenium RC would use a
-proxy server and inject java script in browser to be able to control
-it. Given the intrusive nature Selenium RC had on browser,
-you were never sure if you are pushing the same application live what
-you tested. Selenium 2 APIs yet contain Selenium RC APIs but
-Selenium 3 would completely get rid of Selenium RC APIs. If you are
-still using Selenium RC then you must
-<em><a href=rctowd.html>migrate</a></em> to Selenium WebDriver.
+<p> Soon after development of WebDriver tests you may face need of
+ running your test on multiple combinations of browser/operating
+ system. This is where <em><a href=grid.html>Selenium Grid</a></em>
+ comes to rescue.
 
 
 <h2>Selenium IDE</h2>
 
 <p><em><a href=ide.html>Selenium IDE</a></em>, is a
-firefox plugin which can be used to record test steps on a firefox
-browser. Selenium IDE can be used to generate <i>quick and dirty</i>
+ firefox plugin which can be used to record test steps on a firefox
+ browser. Selenium IDE can be used to generate <i>quick and dirty</i>
  test code and convert in a variety of programming languages (i.e. c#,
-java, python and ruby). Given the maintainability of code generated
-through Selenium IDE, it is not recommended to use it for any thing
-more than getting acquainted with element locators or generating
-<i>throw away code</i>. We are sure that once you get used to
-WebDriver API then you will never use Selenium IDE.
+ java, python and ruby). Given the maintainability of code generated
+ through Selenium IDE, it is not recommended to use it for any thing
+ more than getting acquainted with element locators or generating
+ <i>throw away code</i>. We are sure that once you get used to
+ WebDriver API then you will never use Selenium IDE.
 
 
-<h2>Selenium Grid</h2>
+<h2>Selenium Remote Control</h2>
 
-<p> Soon after development of WebDriver tests you may face need of
-running your test on multiple combinations of browser/operating
-system. This is where <em><a href=grid.html>Selenium Grid</a></em>
-comes to rescue.
+<p><em>Selenium Remote Control</em>, is also known
+ as Selenium 1. Selenium RC was the most prominent Selenium tool
+ before the advent of Selenium WebDriver. Selenium RC would use a
+ proxy server and inject java script in browser to be able to control
+ it. Given the intrusive nature Selenium RC had on browser,
+ you were never sure if you are pushing the same application live what
+ you tested.
+
+<p>It is currently possible to run older RC tests using Selenium 3, by
+ including the <em>leg-rc</em> package in your project, which attempts
+ to provide an RC like API implemented with WebDriver. It is likely that
+ leg-rc will be discontinued in the future, so efforts should be made to
+ <a href=rctowd.html>migrate</a> fully to WebDriver. It is highly
+ unrecommended to start any new project using Selenium RC currently.
```

Comment:
Best phrase things positively: `it is strongly recommended that new projects use the WebDriver APIs`.

The reason being people tend to gloss over words like "not" for some reason.